### DISCUSSION TITLE
TE3

### DISCUSSION THREAD

**Discussion by BoluxByte** [web-classroom/web-classroom.github.io#24] [Q&A : Questions organisationnelles]
Coucou Monsieur,

Je souhaiterais savoir si vous pourriez nous donner des précisions concernant le format du TE3 comme promis ce mardi afin qu'on puisse avoir une préparation optimale ce week-end.

Meilleures salutations,
Pablo Saez

**Comment by OlivierLmr** [comment]
Bonjour,

Le test vous demandera d'implémenter un petit projet de A à Z (avec peut-être quelques petites choses fournies, pour accélérer le processus). Il vous faudra donc utiliser les grands thèmes vus ce semestre (HTML, CSS, JS avec POO et async, ce genre de choses) en cours et en labos. Mon but sera d'évaluer votre capacité à implémenter une application web type, nécéssitant les fonctionnalités les plus communément utilisées sur le Web.

Pour les 3 derniers chapitres (surtout react et typescript), il faudra que vous ayez compris ce qu'ils sont, ce qu'ils permettent, mais pas le détail de leur fonctionnement (sinon ça deviendrait des questions de cours et ça ne m'intéresse pas vraiment). Donc normalement pas besoin d'y passer trop de temps. Le coeur du test sera ce projet.

Si vous avez d'autres questions j'y réponds avec plaisir.

À mardi

**Reply by AlexisMts** [reply]
Bonjour,

Si je comprends bien le format s'approche de celui des laboratoires avec un travail plutôt pratique ?
Est-ce qu'on aura donc le droit à la documentation du cours, ainsi que l'accès à Internet pour nos recherches ?

Bonne journée

**Reply by OlivierLmr** [reply]
C'est ça, on peut voir ça comme un petit labo noté à faire seul sur une durée limitée (encore à déterminer), dans un contexte réaliste, c'est à dire avec accès à tout (Google ou équivalent, Copilot, etc), simplement pas de communication entre vous ou avec d'autres humains.
